channeldb/payments.go

```diff
@@ -17,8 +17,40 @@ var (
 	// which is a monotonically increasing uint64.  BoltDB's sequence
 	// feature is used for generating monotonically increasing id.
 	paymentBucket = []byte("payments")
+
+	// paymentStatusBucket is the name of the bucket within the database that
+	// stores all data related to relation between payment hash and
+	// payment status
+	paymentStatusBucket = []byte("payment-status")
 )
 
+// PaymentStatus represent current status of payment
+type PaymentStatus uint8
+
+const (
+	// StatusGrounded is status where payment is initiated and received
+	// an intermittent failure
+	StatusGrounded PaymentStatus = iota
```

Comment:
Since these values are being persisted, we should assign concrete integers to make sure that ordering doesn't break anything. 0, 1, 2 should be fine